lib/puppet/file_system/windows.rb

```diff
@@ -3,6 +3,16 @@
 
 class Puppet::FileSystem::Windows < Puppet::FileSystem::Posix
 
+  def path_string(path)
```

Comment:
I'm not sure that using `path_string` here is the best... I think it might be better to add a new method to file_system like path_string... or perhaps just use `expand_path` where necessary.  Going to try an `expand_path` variant.